```
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   TestBalancer.testBalancerCliWithExcludeList:1462->doTest:808->doTest:939->runBalancerCli:1100->waitForBalancer:674
[ERROR]   TestBalancer.testBalancerCliWithExcludeListInAFile:1494->doTest:808->doTest:939->runBalancerCli:1100->waitForBalancer:674
[ERROR]   TestBalancer.testBalancerCliWithIncludeListInAFile:1584->doTest:808->doTest:939->runBalancerCli:1100->waitForBalancer:679
[ERROR]   TestBalancer.testBalancerWithExcludeList:1431->doTest:808->doTest:941->runBalancer:978->waitForBalancer:674
[ERROR]   TestBalancer.testBalancerWithIncludeList:1524->doTest:808->doTest:941->runBalancer:978->waitForBalancer:679
[ERROR]   TestBalancer.testBalancerWithIncludeListWithPorts:1539->doTest:808->doTest:941->runBalancer:978->waitForBalancer:679
[ERROR]   TestBalancer.testBalancerWithSortTopNodes:2312 expected:<900> but was:<950>
[ERROR] Errors: 
[ERROR]   TestDFSInotifyEventInputStreamKerberized.testWithKerberizedCluster:116 ? IO De...
[ERROR]   TestBalancer.testBalancerWithStripedFile:2068->doTestBalancerWithStripedFile:2121->runBalancer:975->waitForHeartBeat:464 Timeout
[ERROR]   TestUnderReplicatedBlocks.testSetRepIncWithUnderReplicatedBlocks:80 ? TestTimedOut
[INFO] 
[ERROR] Tests run: 6706, Failures: 7, Errors: 3, Skipped: 24
```

I think the failed tests are not related to this PR, so I just closed and reopen this PR to trigger another new CI checks.

--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/DFSConfigKeys.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/DFSConfigKeys.java
@@ -393,9 +393,11 @@ public class DFSConfigKeys extends CommonConfigurationKeys {
   public static final String  DFS_NAMENODE_EDITS_ASYNC_LOGGING =
       "dfs.namenode.edits.asynclogging";
   public static final boolean DFS_NAMENODE_EDITS_ASYNC_LOGGING_DEFAULT = true;
-  public static final String  DFS_NAMENODE_EDITS_ASYNC_LOGGING_PENDING_QUEUE_SIZE =
+  public static final String
+      DFS_NAMENODE_EDITS_ASYNC_LOGGING_PENDING_QUEUE_SIZE =
       "dfs.namenode.edits.asynclogging.pending.queue.size";
-  public static final int DFS_NAMENODE_EDITS_ASYNC_LOGGING_PENDING_QUEUE_SIZE_DEFAULT = 4096;
+  public static final int
+      DFS_NAMENODE_EDITS_ASYNC_LOGGING_PENDING_QUEUE_SIZE_DEFAULT = 4096;
 
   public static final String DFS_NAMENODE_PROVIDED_ENABLED = "dfs.namenode.provided.enabled";
   public static final boolean DFS_NAMENODE_PROVIDED_ENABLED_DEFAULT = false;

